### ISSUE TITLE
Combobox: list options are not announced with JAWS and user will not know which item to select

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#2182]
**Describe the bug**
When user select one option from the combobox with keyboard, the selected option cannot be announced sometime by JAWS user. Which will blocking the JAWS users. 

**Isolated Example**
https://codesandbox.io/s/c2fj5?file=/package.json:157-278

**To Reproduce**
1. Go to https://c2fj5.csb.app/
2. Open JAWS
2. Tab to focus the control
3. Using down - narrow key to select the item from top to bottom
4. Selected item cannot be announced sometimes

**Expected behavior**
Selected item should be announced always when selected

**Screenshots**
https://user-images.githubusercontent.com/5724592/135390055-4ec17669-38ad-42cd-853d-d33b8031e1c1.mp4

**UI5 Web Components for React Information**
  "@ui5/webcomponents": "0.31.19",
   "@ui5/webcomponents-fiori": "0.31.19",
   "@ui5/webcomponents-react": "0.131.20",
Operating System: Window 10
Browser: Chrome

**Additional context**
I am from Successfactors, currently, it is a blocking issue.
